turbinia/turbiniactl.py

```diff
@@ -418,6 +423,12 @@ def main():
           "Failed to read config file {0:s}: {1!s}".format(
               config.configSource, exception))
       sys.exit(1)
+  #sends test notification
+  if args.command == 'testnotify':
+    notify.sendmail(
+        config.EMAIL_ADDRESS, 'Turbinia test notification',
+        'This is a test noticiation')
```

Comment:
I should have noticed this one earlier too, but this should be s/noticiation/notification/.